com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/helpers/v2/ConfigurationSubscriptionFactory.java

```diff
@@ -22,40 +23,47 @@
 
     private static final int MTA_MAJOR_SCHEMA_VERSION = 2;
 
-    public List<ConfigurationSubscription> create(DeploymentDescriptor descriptor,
```

Comment:
To have the variables used in methods be thread safe(final) because with the changes introduced here, if that computation was done in the `create` method, it would not have been thread safe